R/datatables.R

```diff
@@ -375,6 +378,9 @@ datatable = function(
     params$autoHideNavigation = autoHideNavigation
   }
 
+  # record lazyRender
+  if(isFALSE(lazyRender)) params$lazyRender = lazyRender
```

Comment:
I think this is enough since `x$foo = NULL` will not create `foo` in `x`.

```suggestion
params$lazyRender = lazyRender
```

Reply:
Makes sense.  Thanks for changing it back.